ts-client/src/dlmm/index.ts

```diff
@@ -4136,17 +4126,22 @@ export class DLMM {
       false
     );
 
-    const initializeBinArraysAndPositionIxs = [];
-    const addLiquidityIxs = [];
-    const appendedInitBinArrayIx = new Set();
+    const seederTokenY = getAssociatedTokenAddressSync(
+      this.lbPair.tokenYMint,
+      owner,
+      false
+    );
+
+    const initializeBinArraysAndPositionIxs: TransactionInstruction[][] = [];
+    const addLiquidityIxs: TransactionInstruction[][] = [];
+    const appendedInitBinArrayIx = new Set<String>();
```

Comment:
any reason to use String instead of PublicKey here?

Reply:
Fixed

Reply:
Reverted, somehow it doesn't work with pubkey